### PR TITLE
fix(assets): add asset configs for chainIds we know + make props optional

### PR DESCRIPTION
**Pull request by LuizAsFight**
make `icon` prop optional
make `description` prop optional
include configs for chainIds / networks

### CODE REVIEW

File: packages/assets/src/types.ts [hunk 1]
```diff
@@ -26,14 +26,12 @@ export type Asset = {
...
   /** description of the asset */
-  description: string;
+  description?: string;
   /** symbol of the asset */
   symbol: string;
   /** icon of the asset */
-  icon: string;
+  icon: string | null;
   /** asset id on Fuel Network */
-  assetId: string;
```
Comment (luizstacio) [inline packages/assets/src/types.ts:35]: This assetId is the main id of the fuel network, this is why is outside of the folder

File: packages/assets/src/types.ts [hunk 2]
```diff
@@ -26,14 +26,12 @@ export type Asset = {
   /** name of the asset */
   name: string;
   /** description of the asset */
-  description: string;
+  description?: string;
   /** symbol of the asset */
   symbol: string;
   /** icon of the asset */
-  icon: string;
+  icon: string | null;
```
Comment (luizstacio) [inline packages/assets/src/types.ts:33]: When an asset is added to this package it's required to contain the icon. If the application not need it we should override the type on the application level.

File: packages/assets/src/types.ts (hunk 1 above)
Comment (LuizAsFight) [reply packages/assets/src/types.ts:35]: the id can be different on different networks

File: packages/assets/src/types.ts (hunk 2 above)
Comment (LuizAsFight) [reply packages/assets/src/types.ts:33]: the problem is with typing. I will make the `resolveIconPath` function force to not be null then.

Comment (luizstacio) [reply packages/assets/src/types.ts:33]: yeah makes sense

File: packages/assets/src/types.ts [hunk 3]
```diff
@@ -26,14 +26,12 @@ export type Asset = {
   /** name of the asset */
   name: string;
   /** description of the asset */
-  description: string;
+  description?: string;
```
Comment (luizstacio) [inline packages/assets/src/types.ts:29]: We can remove this field

**Review by luizstacio — CHANGES_REQUESTED** [review]

**Review by luizstacio — APPROVED** [review]

**Pull request merged**